### PR TITLE
Bump library versions

### PR DESCRIPTION
**Pull request by isengrims**

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.3.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.6-all.zip
```
Comment (Nikita-Smirnov-Exactpro) [inline gradle/wrapper/gradle-wrapper.properties:3]: Please bump in dockerfile too

Comment (isengrims) [reply gradle/wrapper/gradle-wrapper.properties:3]: It is already 7.6 for this branch ( changed in another branch ). Dockerfile is not there ( it is lib repo )

File: src/main/kotlin/com/exactpro/th2/conn/dirty/tcp/core/api/IProtocolHandler.kt [hunk 2]
```diff
@@ -74,6 +74,8 @@ interface IProtocolHandler : AutoCloseable {
      */
     fun onOutgoing(message: ByteBuf, metadata: MutableMap<String, String>) = Unit
 
+    fun getGrpcService()
```
Comment (ivandruzhinin) [inline src/main/kotlin/com/exactpro/th2/conn/dirty/tcp/core/api/IProtocolHandler.kt:77]: It should take service class as a parameter and return service instance. Also why it is in this change?

**Review by ivandruzhinin — CHANGES_REQUESTED** [review]

**Review by ivandruzhinin — APPROVED** [review]
**Please merge with squash. Also don't forget to edit squashed commit message and description while you're at it**

**Review by Nikita-Smirnov-Exactpro — APPROVED** [review]

**Pull request closed without merging**